pypowsybl/glsk.py

```diff
@@ -0,0 +1,30 @@
+#
+# Copyright (c) 2022, RTE (http://www.rte-france.com)
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+#
+import pypowsybl._pypowsybl as _pypowsybl
+from datetime import datetime
+
+class GLSKImporter:
+    def __init__(self, file):
+        self.handle = _pypowsybl.create_glsk_importer(file)
+
+    def get_gsk_time_interval_start(self):
```

Comment:
Instead of get_gsk_time_interval_start and get_gsk_time_interval_end we could get a list of timedate

Reply:
you are right, this is just an interval